app/scripts/fetch_xulrunner

```diff
@@ -483,6 +485,14 @@ if [ $BUILD_LINUX == 1 ]; then
 		if [ -e "$archived_file" ]; then
 			echo "Using $archived_file"
 			cp "$archived_file" "firefox-$GECKO_VERSION.tar.bz2"
+		elif [[ "$arch" == "aarch64" ]]; then
+			curl -O "http://ftp.de.debian.org/debian/pool/main/f/firefox-esr/firefox-esr_102.15.0esr-1~deb11u1_arm64.deb"
+			ar xOf firefox-esr_102.15.0esr-1\~deb11u1_arm64.deb data.tar.xz
+			tar -xf data.tar.xz
+			mv ./usr/lib/firefox-esr ./firefox
+			mv ./firefox/firefox-esr ./firefox/firefox-bin
+			rm -rf ./etc ./usr data.tar.xz firefox-esr_102.15.0esr-1\~deb11u1_arm64.deb
```

Comment:
There are a bunch of symlinks in the archive that will need to be dealt with — either removed or replaced with the real files, depending on what's in the official tarball.